spec/support/helpers/authentication.rb

```diff
@@ -0,0 +1,10 @@
+module Helpers
+  module Authentication
+    def log_in_as(user)
+      visit(new_session_path)
+      fill_in("Email", with: "email@email.com")
+      fill_in("Password", with: "password")
```

Comment:
```suggestion
      fill_in("Password", with: user.password)
```
Im not sure if we can do this one, if not this is fine

Reply:
> Im not sure if we can do this one, if not this is fine

i tried before, doesnt work because of the encryption